packages/forms/frontend/sirius-components-widget-reference/src/ReferenceWidgetFragment.types.ts

```diff
@@ -17,6 +17,7 @@ export interface GQLReferenceWidget extends GQLWidget {
   label: string;
   reference: GQLReference;
   referenceValues: Array<GQLReferenceValue> | null;
+  referenceOptions: Array<GQLReferenceValue> | null;
```

Comment:
Could you use `GQLReferenceValue[] | null` instead for `referenceOptions` and `referenceValues` please?

Reply:
Done